manga-reader.lua

```diff
@@ -5,6 +5,7 @@ local ext = {
 	".avif",
 	".bmp",
 	".cbz",
+	".cbr",
```

Comment:
Mind placing this above `.cbz` real quick?

Reply:
Done